hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsCustomEncryption.java

```diff
@@ -95,53 +110,58 @@ public class ITestAbfsCustomEncryption extends AbstractAbfsIntegrationTest {
   @Parameterized.Parameter(6)
   public boolean isCpkResponseHdrExpected;
 
+  @Parameterized.Parameter(7)
+  public Boolean isCpkResponseKeyExpected = false;
+
+  @Parameterized.Parameter(8)
+  public Boolean fileSystemListStatusResultToBeUsedForOpeningFile = false;
```

Comment:
To test the integration where azureBlobFileSystem.listStatus api response is used as an option for openToReadWithOptions.

Inline with https://github.com/apache/hadoop/pull/3440#discussion_r989075168